plugins/services/src/js/utils/FrameworkUtil.js

```diff
@@ -76,6 +84,62 @@ const FrameworkUtil = {
     }
 
     return splitVersion.slice(1).join("-");
+  },
+
+  /**
+   * Return a warning or an error label
+   * if the pacakge has cosmosPackage been updated soon.
+   * @param {object} cosmosPackage The cosmos package to check.
+   * @param {Boolean} withLabel Indicates whether we want the "Last Updated: " label or not.
+   * @returns {null | String | React.ReactNode} Null or string or an HTML saying when was the last update of this package
+   */
+  getLastUpdated(cosmosPackage, withLabel) {
+    // This looks like magic, but we are only decoding the package configuration into a string,
+    // and then finding the PACKAGE_BUILD_TIME_STR property.
+    const v2AppMustacheTemplate = Util.findNestedPropertyInObject(
```

Comment:
I wonder if we wanna show nothing for packages that don't have this property or should we do "Last Updated: N/A".

Reply:
will clarify with izhar